src/component/contents/DraftEditorContents.react.js

```diff
@@ -146,7 +147,7 @@ class DraftEditorContents extends React.Component {
       };
 
       const configForType = blockRenderMap.get(blockType);
-      const wrapperTemplate = customWrapperTemplate || configForType.wrapper;
```

Comment:
Here we're undoing the overriding of the DraftJS internal wrapper template from a prior pr [here](https://github.com/textioHQ/draft-js/pull/72/files)

Reply:
Why do we want to undo that PR?

Reply:
wrapperTemplate is currently being overloaded by the customWrapperTemplate, but it turns out wrapperTemplate is used internally by DraftJS to format lists and any other elements needing automagic wrapping. This overload led to ol/ul wrapping breaking (https://textio.atlassian.net/browse/EN-1300)

Reply:
My major concern is that we're introducing a new wrapper but it's not clear what that new wrapper is doing or how it's different from the one that exists in Draft already. It makes the change difficult to validate but more importantly it means that we're likely to forget what we were doing when we come back to this in the future and that this code will then become to scary to change.

Sometimes a change like this just needs a large comment that describes the difference, what's happening, and why.